01-Introduction/01-Introduction.ipynb

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
<p>The bullet nesting does not seem to be displaying correctly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I've redone the bulleting. But I'm not sure I addressed the issue. I agree that putting all sub-bullets under archive requirement was really mis-organized</p>

Reply:
<p>I've reorganized the formatting, but if the issue persists (after my next commit) perhaps a screen shot illustrating the issue would help me.</p>